config/eslint-config-carbon/plugins/react.js

```diff
@@ -37,6 +37,9 @@ module.exports = {
     // react-hooks
     'react-hooks/rules-of-hooks': 2,
     'react-hooks/exhaustive-deps': 2,
+
+    // prop-types
+    'react/sort-prop-types': 1,
```

Comment:
```suggestion
```

Should be configured as an error now after the prop type refactor!

Reply:
Fixed 👍